### DISCUSSION TITLE
How to create event handlers via conductor ui?

### DISCUSSION THREAD

**Discussion by viinxent** [Netflix/conductor#3215] [Q&A] [answered]
Conductor UI event tab doesn't have a create event button.

**Comment by peterlau** [comment, marked as answer]
The UI currently does not support creating event handlers. You will need to use the API (specifically, the POST `/event` endpoint) directly or via the Swagger page.

This feature may become prioritized in the future, and as always, if you would like to help contribute code, please let us know!

**Reply by viinxent** [reply]
Didn't know about swagger page, will check on that one, thanks a lot!
